release-notes.txt

```diff
@@ -1,3 +1,230 @@
+RIOT-2015.12 - Release Notes
+============================
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things: 8-bit
+microcontrollers, 16-bit microcontrollers and light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, real-time
+capabilities, small memory footprint, modularity, and uniform API access,
+independent of the underlying hardware (this API offers partial POSIX
+compliance).
+
+RIOT is developed by an international open source community which is
+independent of specific vendors (e.g. similarly to the Linux community).
+
+About this release:
+============
+This release is mostly a clean-up and bug-fixing release. Besides that, it introduces SAUL,
+the [S]ensor [A]ctuator [U]ber [L]ayer, which offers a unified API to interact with all
+different types of sensors and actuators on RIOT supported hardware. Furthermore, it re-enables
+the support for ICN by integrating CCN-Lite as a package. A lot of new overall documentation was
+added and existing documentation was improved. In addition, a Vagrant (https://www.vagrantup.com/)
+configuration file was added to the RIOT repository in order to create reproducible and
+portable environments that contain all necessary toolchains.
+
+About 222 pull requests with about 631 commits have been merged since the last release and 48
+additional issues have been solved. 37 people contributed code in 102 days. 980 files have been
+touched with ~59779 insertions and ~12115 deletions.
+
+Notations used below:
+============
++ means new feature/item
+* means modified feature/item
+- means removed feature/item
+
+
+New features and changes
+============
+
+General
+-----------
+
+Device support
+---------------------
++ SAUL [S]ensor [A]ctuator [U]ber [L]ayer
+
+Core
+------
+- Removed dINT()/eINT() and replaced them with disableIRQ()/enableIRQ()/restoreIRQ()
```

Comment:
maybe instead:
`* replaced deprecated dINT()/eINT() calls by up-to-date disableIRQ()/enableIRQ()/restoreIRQ() calls throughout the whole core`
or something similar.